packages/cli/README.md

```diff
@@ -104,10 +105,10 @@ See examples/module-design for such a design
 
 - or run the CLI on the main file if you know which one it is
 
-```openjscad <PATH TO DESIGN FOLDER>/index.js <OPTIONS>```
+```openjscad <PATH TO DESIGN FOLDER>/main.js <OPTIONS>```
```

Comment:
why the change to main ? we want to make people have good practices, and we are in js, so entry points are usually index.js (just to be clear, it may seem like I am nitpicky, but I see no reason to change something that is not wrong)

Reply:
I really don’t know what’s correct. I’d be glad to add the hierarchy.

Reply:
Reverted. Now says index.js